packages/renderer/src/components/molecules/WalletBalance/WalletBalance.tsx

```diff
@@ -25,10 +25,21 @@ import {
 } from "@mantine/core";
 import { ReactComponent as XMRLogo } from "@assets/xmr-logo-1.svg";
 import { ReactComponent as ArrowDown } from "@assets/arrow-down.svg";
+import { useBalances } from "@hooks/haveno/useBalances";
+export interface balances {
```

Comment:
Code style: interface names should be PascalCased. Why is this being exported? I don't even see this being used anywhere.